proto/tendermint/abci/types.proto

```diff
@@ -282,6 +300,21 @@ message ResponseApplySnapshotChunk {
   }
 }
 
+message ResponsePrepareProposal {
+  tendermint.types.Data block_data = 1;
+}
+
+message ResponseProcessProposal {
+  Result         result   = 1;
+  repeated bytes evidence = 2;
```

Comment:
What is the semantics of this field? (and why isn't it of the `Evidence` type defined in this file?)
I checked the usage of this structure in `state/execution.go` and it is not used.